docs/FAQ.md

```diff
@@ -8,15 +8,12 @@ menu:
     weight: 9
     identifier: "operator-faq"
 aliases:
-  - /operator/FAQ.html
-  - /operator/faq.html
+  - /operator/faq/
+  - /operator/faq/index.html
 ---
-
-# FAQ (Frequency Asked Questions)
-
 ## How do you monitor the operator itself?
 
-You can read about vmoperator monitoring in [this document](./monitoring.md).
+You can read about vmoperator monitoring in [this document](./monitoring/).
```

Comment:
But those relative paths are not valid on github, no?

Reply:
you can run `make docs-debug` and check it at http://localhost:1313

Reply:
@Haleygo fixed links, now they are working fine for both github and for vmdocs